dips/dip-1.mdx

```diff
@@ -19,7 +19,7 @@ The Off-Chain Protocol allows two entities to define payments off-chain and priv
 | Version | Revision | Link      |
 |---------|----------|-----------|
 | v2      | current  | n/a       |
-| v1      | e82a0eb8a9dd5d498fc89bf84565cc7adae3d8ef | https://github.com/diem/dip/blob/e82a0eb8a9dd5d498fc89bf84565cc7adae3d8ef/dips/dip-1.mdx  |
+| v1      | e82a0eb8a9dd5d498fc89bf84565cc7adae3d8ef | https://github.com/diem/lip/blob/e82a0eb8a9dd5d498fc89bf84565cc7adae3d8ef/lips/lip-1.mdx  |
```

Comment:
I don't think there is 'lip' only 'dip'

Reply:
Try the links, this was an automated change that broke it. This is pointing to the repo when we still called them lips.

Reply:
I guess there are 2 parts then, one is codebase name, which is dip, and lip will be redirected to dip, the following part of file names should be lips.
Im just wondering whether the redirect will not work for the codebase name in the future.

Reply:
oh that's a good call, I can probably change the "lip/blob" to "dip/blob" we're abusing the redirect on that case